lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1094,7 +1098,7 @@ sub _create_bam_index {
     return 1;
 }
 
-sub _create_bam_flagstat {
+sub create_bam_flagstat {
```

Comment:
We debated this, added!

My general thinking here is that I do not check for file existence before locking because we should not be dying here in general, so I'm not concerned about checking before locking.